packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -91,6 +91,8 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
                     }
                     this._buildIdentifier = capability['browserstack.buildIdentifier']?.toString()
                     this._buildName = capability['build']?.toString()
+                    // @ts-ignore
+                    this._projectName = capability['project']?.toString()
```

Comment:
Why ts-ignore?

Reply:
Property 'project' does not exist on type 'DesiredCapabilities'.